amazon-kinesis-client/src/main/java/software/amazon/kinesis/retrieval/polling/PollingConfig.java

```diff
@@ -40,6 +40,8 @@ public class PollingConfig implements RetrievalSpecificConfig {
 
     public static final Duration DEFAULT_REQUEST_TIMEOUT = Duration.ofSeconds(30);
 
+    public static final int DEFAULT_MAX_RECORDS = 10000;
```

Comment:
If this is going to be public and static, we can share this value in the multilang as well so that there is one central source of truth.

Reply:
as a follow up, it may be worth checking to see how many of these types of values we have "copy pasted" in multiple locations and see if its worth centralizing them.